### PR TITLE
Add document page setup and create document collection type in strapi

### PR DESCRIPTION
**Pull request by AlexNisavic1**

### CODE REVIEW

File: izadji/src/App.tsx [hunk 1]
```diff
@@ -71,6 +72,14 @@ function App() {
               <ProjectPage />
             </Layout>
           }
+        />
+         <Route
+          path={routes.documentatio}
```
Comment (DusicaVasicFax) [inline izadji/src/App.tsx]: Spelling mistake its documentation

File: izadji/src/pages/DocumentationPage/DocumentationPage.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import { ReactNode, useEffect, useState } from 'react';
+
+import 'swiper/scss';
+import 'swiper/scss/pagination';
+import documentationPageService from '../../services/documentationService';
+import { MapComponents } from '../../utils/mappers/sharedMapper';
+import Publications from '../../components/Publications/Publications';
+import { PublicationProps } from '../../components/Publication/Publication';
+
+const GetPublications = (publication: any): PublicationProps => {
```
Comment (DusicaVasicFax) [inline izadji/src/pages/DocumentationPage/DocumentationPage.tsx]: I suggest renaming this property to documentation and props to DocumentationProps

File: izadji/src/pages/DocumentationPage/DocumentationPage.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        download:publication.attributes.downloadButton.pdfFile.data.attributes.hash,
+    };
+};
+
+const DocumentationPage = () => {
+  const [components, setComponents] = useState<ReactNode[]>([]);
+
+  useEffect(() => {
+    documentationPageService.getDocumentationPage().then((response) => {
+      documentationPageService.getDocuments().then(publicationsResponse=> {
```
Comment (DusicaVasicFax) [inline izadji/src/pages/DocumentationPage/DocumentationPage.tsx]: Also renaming for this response

File: izadji/src/pages/DocumentationPage/DocumentationPage.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        MapComponents(
+          response.data.data.attributes.components,
+          [],setComponents,
+          {
+            appendBefore: [
+              <Publications
```
Comment (DusicaVasicFax) [inline izadji/src/pages/DocumentationPage/DocumentationPage.tsx]: Are we using the same component for Publication and Documentation, if we are can we name it somehow different so we know it applies to both of the pages?

File: izadji/src/routes.ts [hunk 5]
```diff
@@ -9,7 +9,8 @@ export const routes = {
   project: '/project/:id',
   aboutUs: '/about-us',
   donate: '/donate',
-  publication: '/publications'
+  publication: '/publications',
+  documentatio: '/documentation'
```
Comment (LazarJ98) [inline izadji/src/routes.ts]: Think you have a typo here, missing letter **n**.

Comment (AlexNisavic1) [reply izadji/src/routes.ts]: fixed it, thanks for catching this.

File: izadji/src/pages/DocumentationPage/DocumentationPage.tsx (hunk 2 above)
Comment (EpicGuy4000) [inline izadji/src/pages/DocumentationPage/DocumentationPage.tsx]: Bit weird to use publicationProps for documentation, even if they are the same.
We should either dupli ate the props, so they change separatly, or extract them (and rename them) so we can reuse them.

Comment (AlexNisavic1) [reply izadji/src/pages/DocumentationPage/DocumentationPage.tsx]: Agreed, extracted it to DownloadableProps

**Review by EpicGuy4000 — APPROVED** [review]

File: izadji/src/pages/DocumentationPage/DocumentationPage.tsx [hunk 6]
```diff
@@ -0,0 +1,46 @@
+import { ReactNode, useEffect, useState } from 'react';
+
+import 'swiper/scss';
+import 'swiper/scss/pagination';
+import documentationPageService from '../../services/documentationService';
+import { MapComponents } from '../../utils/mappers/sharedMapper';
+import Publications from '../../components/Publications/Publications';
+import DownloadableProps from '../../utils/DownloadableProps';
+
+const GetPublications = (publication: any): DownloadableProps => {
```
Comment (DusicaVasicFax) [inline izadji/src/pages/DocumentationPage/DocumentationPage.tsx]: This should be GetDocumentation

**Review by DusicaVasicFax — CHANGES_REQUESTED** [review]

File: izadji/src/App.tsx (hunk 1 above)
Comment (AlexNisavic1) [reply izadji/src/App.tsx]: This was fixed in the last commit.

File: izadji/src/pages/DocumentationPage/DocumentationPage.tsx (hunk 2 above)
Comment (AlexNisavic1) [reply izadji/src/pages/DocumentationPage/DocumentationPage.tsx]: This was also changed in the last commit, please review again.

**Review by DusicaVasicFax — APPROVED** [review]

**Pull request merged**